docker/Dockerfile

```diff
@@ -20,10 +20,12 @@ WORKDIR /root/
 
 # Install dependencies
 RUN apk update
-RUN apk --no-cache add apache-ant bash git openjdk11
+RUN apk --no-cache add apache-ant bash git openjdk11 supervisor
 
+# Establish environment variables
 RUN echo 'export JAVA_HOME=/usr/lib/jvm/java-11-openjdk' >> $HOME/.bashrc
-env NUTCH_HOME='/root/nutch_source/runtime/local'
+ENV JAVA_HOME '/usr/lib/jvm/java-11-openjdk'
```

Comment:
The [Dockerfile reference](https://docs.docker.com/engine/reference/builder/#env) discourages the use of `ENV var val` in favor of `ENV var=val`. I've seen this variant the first time and had to try out and read the documentation.

+1 to unify how JAVA_HOME and NUTCH_HOME are set. Also good not to force the usage of the bash as command-line shell.

Reply:
Thanks @sebastian-nagel